testing.md

```diff
@@ -1,348 +0,0 @@
-# Testing Conventions
```

Comment:
These conventions were never reinforced. Argument is to not keep it; perhaps redefine our testing conventions.